tests/test_photon_ops.py

```diff
@@ -596,12 +649,15 @@ def test_config_rubin_optics():
                 },
             ]
         },
+        "sky_pos": create_test_img_wcs(
+            boresight=galsim.CelestialCoord(1.1047934165124105 * galsim.radians, -0.5261230452954583 * galsim.radians),
+            rottelpos=np.pi / 3 * galsim.radians
+        ).toWorld(image_pos),
```

Comment:
Slightly cleaner here would be to define boresight once outside the config, and then use both `"boresight": boresight` above in photon_ops and use that same value here for the `boresight` parameter.  Otherwise there are two places where a bunch of random numbers need to be visually confirmed to match up.

Reply:
Thanks Mike, nice spot! I've made that change now.